brightchain-lib/src/lib/services/blockService.ts

```
@@ -0,0 +1,796 @@
+      );
+      if (createECBL) {
+        return new ExtendedCBL(
+          blockSize,
+          creator,
+          fileSize,
+          blockIDs,
+          fileName,
+          mimeType,
+        );
+      } else {
+      }
```

ESLint

[error] Empty block statement.

(no-empty)